src/client/pages/Debugger/components/Offer.tsx

```diff
@@ -2,7 +2,13 @@ import React, { useEffect, useState } from 'react'
 import { Form, Formik, FormikProps } from 'formik'
 import { v4 as uuid } from 'uuid'
 import { colorsV3 } from '@hedviginsurance/brand'
-import { CreateQuoteInput, useQuoteLazyQuery } from 'data/graphql'
+import { CreateQuoteVariables } from '@hedviginsurance/embark'
+import {
+  CreateQuoteInput,
+  useCreateDanishHomeAccidentQuoteMutation,
+  useCreateDanishHomeAccidentTravelQuoteMutation,
```

Comment:
Actually, now when I think about it I'm wondering if we rally should create these mutations that will only be used in the debugger? The way I think it works with bundles for Norway in the real onboarding Embark flow is that the `createQuote` mutation is just used twice - once for each contract. And if that's the way we'll do it in Denmark (which I guess it is but please correct me if I misunderstood) then maybe the debugger should mirror that? In the way that we should use the same mutation that will be used in the real flow, I mean. That way we can also be more confident that it will work with Embark as well, or? What are your thoughts on that everyone?

Reply:
I have the least insight I think on this so I might be completely off 😄, but didn't we say that DK should have separate mutations because the way we did bundles in NO was not the path we wanted to continue on?

Reply:
Oh, that might be completely true, I just missed slash forgot that then, sorry about that! So does that mean that these mutations will be used in Embark as well?

Reply:
I think it's more readable to to one mutation instead of multiple and more easily managed. But I agree 100% with your point of being consistent with how we handle it in Embark/Onboarding.